frontend/src/pages/ContractTest.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import '../styles/App.scss';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import Web3Login from '../components/Web3Login';
+import { BooleanStatus } from '../components/Icons';
+import Eth from 'ethjs-query';
+import EthContract from 'ethjs-contract';
+import { BattleGroup } from './../contracts';
+
+class DebugPage extends Component {
+  test = senderAddress => {
+    const eth = new Eth(window.web3.currentProvider);
+    const contract = new EthContract(eth);
+
+    const miniToken = contract(BattleGroup.ABI).at(BattleGroup.address);
+
+    miniToken
+      .createBattleGroup(senderAddress, [4, 3, 4, 2, 3], {
```

Comment:
No battlegroup should ever be created containing duplicate cards.

Reply:
this is test data lol

Reply:
Just just use non-duplicate id's. Even `[1, 2, 3, 4, 5]`